.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     - name: Docker dependencies
       if: ${{ matrix.os == 'ubuntu-latest' }}
       run: |
-        docker-compose up -d
+        docker compose up -d
```

Comment:
could you also update the `release.yml` file?

Reply:
the release has already been updated
https://github.com/ArtemIsmagilov/emcache/blob/f4053c88bb3d38f540f28215b87b8b2a1b18cc92/.github/workflows/release.yml#L92